src/components/TodoInfo/TodoInfo.tsx

```
@@ -1 +1,18 @@
-export const TodoInfo = () => {};
+import { UserI } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+  user: UserI | undefined;
+}
+
+export const TodoInfo = ({ todo, user }: Props) => {
+  return (
+    <>
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      <a className="UserInfo" href={user?.email}>
+        {user?.name}
```

The component uses optional chaining for `user` (`user?.email` and `user?.name`). This masks missing data instead of failing fast; it ties into the same checklist violation above. After making `user` required you can render `href` and content directly (and consider using `href={`mailto:${user.email}`}` to make the link an email link).